CVLByExample/ERC20/README.md

```diff
@@ -9,8 +9,27 @@ the following rules fail:
 `balancesBoundedByTotalSupply` - fails for functions `burn`, `deposit`, and `mint`.
 `totalSupplyIsSumOfBalances` - fails for functions `deposit` and `withdraw`.
 
+These rules fail for the following reasons:
+1. `deposit` updates `_balances[msg.sender]` but does not update `_totalSupply` accordingly.
+2. `withdraw updates `_balances[msg.sender]` but does not update `_totalSupply` accordingly.
+
+## Incorrect Spec
+1. The function `transferFrom(address,address,uint)` is defined in the method block of `ERC20.spec` as
+   `envfree` but it refers to `env` so cannot be defined as such.
+2. The invariant `balancesBoundedByTotalSupply` allows the receiver and sender in `transferFrom` to be the same.
+3. The invariant `balancesBoundedByTotalSupply` checks the balances of `alice` and `bob` and therefore assumes
```

Comment:
basically balancesBoundedByTotalSupply is a wrong rule and is covered by the sumof balances, maybe we should remove it in the fixed and change it to a cvl function that assumes this and it is a safe assumption

Reply:
remove balancesBoundedByTotalSupply and add a cvl function that assumes what?
and where is the cvl function used?